gpkit/nomials/variables.py

```diff
@@ -121,8 +121,10 @@ def __new__(cls, shape, *args, **descr):
                 descr["label"] = arg
 
         values = []
-        for value_option in ["value", "sp_init"]:
-            if value_option in descr:
+        value_option = None
+        for val_option in ["value", "sp_init"]:
+            if val_option in descr:
+                value_option = val_option
```

Comment:
I am dropping this commit -- this is now the second alternative proposed change to fix this issue (the first being 419ff866eda5e861266182c2177ef30e919442f8). Need @bqpd to review intent here before we disable the warning.